.github/workflows/ci.yaml

```diff
@@ -0,0 +1,17 @@
+name: CI
```

Comment:
Where does this actually run?

Reply:
on GitHub's runners! it's free for public repositories. https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions